pkg/chunk/cache/memcached.go

```diff
@@ -235,14 +235,13 @@ func (c *Memcached) Store(ctx context.Context, keys []string, bufs [][]byte) {
 }
 
 // Stop does nothing.
-func (c *Memcached) Stop() error {
+func (c *Memcached) Stop() {
 	if c.inputCh == nil {
```

Comment:
Stylistic thing but maybe simplify this and convert it to != nil and put the logic in there instead of the if and return and then the part we care about.

Reply:
The Go way is to return early and do the work outside of an `if`.